server/config/config.go

```diff
@@ -307,6 +307,10 @@ func GetConfig(v *viper.Viper) Config {
 	}
 }
 
-func (c Config) ValidateBasic() bool {
-	return c.BaseConfig.MinGasPrices != ""
+func (c Config) ValidateBasic() error {
+	if c.BaseConfig.MinGasPrices == "" {
```

Comment:
Throwing an error, if MinGasPrices is empty. So app developer sets some value using `app.toml`.

Reply:
can the app developer override the default value?

Reply:
yes. There are two options:
1. Using flag `./build/simd start --minimum-gas-prices  0.25token1` yields no error.
2. Using env var ` MINIMUM_GAS_PRICES=0.0001token2 ./build/simd start` does not work . It outputs an error : `Error: Please set min gas price in app.toml or flag or env var`. Does anyone has idea why?

Reply:
oh sorry, i meant as the app developer i put a value in my app that automatically sets this.

Reply:
try `MIN_GAS_PRICES=0.0001token2` maybe?

Reply:
I thought it had to be prefixed by the app name? e.g. `SIMAPP_*`?

Reply:
ah yeah, it seems you're right. The serverCtx (used in this PR) needs the app prefix:
https://github.com/cosmos/cosmos-sdk/blob/9fd866e3820b3510010ae172b682d71594cd8c14/server/util.go#L121
but the clientCtx doesn't (at least in simapp):
https://github.com/cosmos/cosmos-sdk/blob/d7dd1d7affc02a819845802da540bc2742d82074/simapp/simd/cmd/root.go#L50

😢 https://github.com/cosmos/cosmos-sdk/issues/9034

Reply:
> I thought it had to be prefixed by the app name? e.g. `SIMAPP_*`?

 `SIMD_MINIMUM_GAS_PRICES=0.0001token2 ./build/simd start`  WORKS and outputs NO error.